### PR TITLE
New serverless pattern - s3-object-lambda-textract

### PR DESCRIPTION
**Pull request by sonihemish**
*Issue #, if available:*

*Description of changes:*
Added a new pattern. S3 Object Lambda to call Textract to return extracted of a document. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by sonihemish** [comment]
updated example json

**Comment by sonihemish** [comment]
also you can contact me internally on slack @hnsoni

File: s3-object-lambda-textract/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# AWS S3 Object Lambda to Amazon textract
+
+```
+  _________________   ________ ___.        __               __    .____                  ___.        .___          __           ___________              __                        __   
+ /   _____/\_____  \  \_____  \\_ |__     |__| ____   _____/  |_  |    |   _____    _____\_ |__    __| _/____    _/  |_  ____   \__    ___/___ ___  ____/  |_____________    _____/  |_ 
+ \_____  \   _(__  <   /   |   \| __ \    |  |/ __ \_/ ___\   __\ |    |   \__  \  /     \| __ \  / __ |\__  \   \   __\/  _ \    |    |_/ __ \\  \/  /\   __\_  __ \__  \ _/ ___\   __\
+ /        \ /       \ /    |    \ \_\ \   |  \  ___/\  \___|  |   |    |___ / __ \|  Y Y  \ \_\ \/ /_/ | / __ \_  |  | (  <_> )   |    |\  ___/ >    <  |  |  |  | \// __ \\  \___|  |  
+/_______  //______  / \_______  /___  /\__|  |\___  >\___  >__|   |_______ (____  /__|_|  /___  /\____ |(____  /  |__|  \____/    |____| \___  >__/\_ \ |__|  |__|  (____  /\___  >__|  
+        \/        \/          \/    \/\______|    \/     \/               \/    \/      \/    \/      \/     \/                              \/      \/                  \/     \/
```
Comment (mavi888) [inline s3-object-lambda-textract/README.md:9]: this is extra please remove

File: s3-object-lambda-textract/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                  |                                        +----------v-----------+                                     |
+                  |                                        |                      |                                     |
+                  |                                        |                      |                                     |
+                  |                                        |    Amazon Textract   |                                     |
+                  |                                        |                      |                                     |
+                  |                                        |                      |                                     |
+                  |                                        +----------------------+                                     |
+                  |                                                                                                     |
+                  +-----------------------------------------------------------------------------------------------------+
```
Comment (mavi888) [inline s3-object-lambda-textract/README.md:111]: use an image not text for this

**Comment by mavi888** [comment]
checks failing and main big issues in the read me. I will review this in detail when those issues get fixed.

**Comment by sonihemish** [comment]
Changes made kindly review

File: s3-object-lambda-textract/README.md (hunk 2 above)
Comment (sonihemish) [reply s3-object-lambda-textract/README.md]: Done

File: s3-object-lambda-textract/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Step Functions to Athena",
```
[the remark this answers is not in the document]
Comment (sonihemish) [reply s3-object-lambda-textract/example-pattern.json]: Done

File: s3-object-lambda-textract/README.md (hunk 1 above)
Comment (sonihemish) [reply s3-object-lambda-textract/README.md]: Done

**Comment by mavi888** [comment]
There are still some issues with the template file, please validate it

**Comment by sonihemish** [comment]
changes made. please review

**Comment by mavi888** [comment]
Thanks for submitting your pattern.
Now this pattern is public in Serverless land

https://serverlessland.com/patterns/s3-object-lambda-textract

You can share your pattern using this link :)

**Pull request merged**